wasm/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"strings"
+	"syscall/js"
+
+	"github.com/centrifuge/precise-proofs/examples/documents"
+	"github.com/centrifuge/precise-proofs/proofs"
+	"github.com/centrifuge/precise-proofs/proofs/proto"
+	"github.com/golang/protobuf/jsonpb"
+)
+
+func checkErr(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func generateProof(i []js.Value) {
```

Comment:
Probably you will already do that in following commits, but we need to take a list of proof fields ([]string) in conjunction with the document, or that will be wrapped within the `[]js.Value` as other items in the list?

Reply:
@mikiquantum let's keep the example simple and make it just create a proof for one field instead of multiple.